### DISCUSSION TITLE
Will this work with Github Actions?

### DISCUSSION THREAD

**Discussion by tking16** [SeleniumHQ/docker-selenium#2313] [Q&A] [answered]
I have a plan to use docker-selenium for my Android tests. In my mind I'll have Selenium Grid running in it's own container on one single job and have the nodes and tests all running on another job. Will that method work though? As I know Github Actions will kill a process as soon as it's job is completed, so I guess I won't be able to have an isolated job just for the Grid?

<img width="268" alt="Screenshot 2024-07-19 at 15 21 11" src="https://github.com/user-attachments/assets/fa4c6666-3127-41a3-8662-979feadd8fae">

**Comment by hikouki-gumo** [comment, marked as answer]
Is Standalone mode work for your use case?

**Reply by tking16** [reply]
Yes perhaps, as that's what I've been using for iOS. Thanks!
